lib/atlas/scaler/graph_exporter.rb

```diff
@@ -0,0 +1,37 @@
+module Atlas
+  class Scaler::GraphExporter
```

Comment:
Probably this whole class should be a parent / sibling class of Atlas::Exporter, as [suggested by @antw](https://github.com/quintel/internal/issues/5#issuecomment-265120551) ([also here](https://github.com/quintel/internal/issues/5#issuecomment-265103928))